clients/python/docs/source/conf.py

```diff
@@ -0,0 +1,61 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "Kaskada Python Client"
+copyright = "2023, The Kaskada Authors"
+author = "The Kaskada Authors"
+release = "0.1.2"
+
+# Path setup
+import sys
+import os
+
+sys.path.insert(0, os.path.abspath("../../src/"))
+
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx_autodoc_typehints",
+    "sphinx.ext.autosummary",
+    "sphinx.ext.coverage",
+    "sphinx.ext.napoleon",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.viewcode",
+    "sphinx_rtd_theme",
+]
+# autosummary_generate = True  # Turn on sphinx.ext.autosummary
+
+autosummary_generate = True  # Turn on sphinx.ext.autosummary
+autoclass_content = "both"  # Add __init__ doc (ie. params) to class summaries
+html_show_sourcelink = (
+    False  # Remove 'view source code' from top of page (for html, not python)
+)
+autodoc_inherit_docstrings = True  # If no docstring, inherit from base class
+set_type_checking_flag = True  # Enable 'expensive' imports for sphinx_autodoc_typehints
+# autodoc_typehints = "description" # Sphinx-native method. Not as good as sphinx_autodoc_typehints
+add_module_names = False  # Remove namespaces from class/method signatures
+
+# master_doc = "index"
+
+# Mappings for sphinx.ext.intersphinx. Projects have to have Sphinx-generated doc! (.inv file)
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3/", None),
+}
+
+templates_path = ["_templates"]
+exclude_patterns = []
```

Comment:
@therapon You may be running into the issue where we are trying to generate the docs on the protobuf generated code. I'd suggest removing or adding an exclude for: `kaskada/kaskada`. The generated python protobuf code goes in `clients/python/src/kaskada/kaskada`